src/connect.js

```diff
@@ -20,7 +20,10 @@
  * @return {Function}
  */
 function connect(params) {
-  // write code here
+  return extraParams => {
+    const givenExtraParams = Object.assign({}, params, extraParams);
```

Comment:
use spread operator and you don't need to write `return` for inner function, there is not so much of code